sdk_extensions/logging/support/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id "java"
+    id "maven-publish"
+
+    id "ru.vyarus.animalsniffer"
+}
+
+description = 'OpenTelemetry Contrib Logging Support'
+ext.moduleName = "io.opentelemetry.extensions.logging.support"
```

Comment:
This should be "io.opentelemetry.sdk.extensions.logging" and all the classes should be in this package or subpackages of this package